corehq/apps/builds/templates/builds/edit_menu.html

```diff
@@ -140,6 +152,7 @@ <h3 class="panel-title">Menu Options</h3>
                             <th>Version</th>
                             <th>Label</th>
                             <th>Superuser Only</th>
+                            <th>J2me Enabled</th>
```

Comment:
these should also all be translated (looks like an existing issue for the others)